src/Serilog/Capturing/PropertyValueConverter.cs

```diff
@@ -165,7 +175,15 @@ LogEventPropertyValue CreatePropertyValue(object? value, Destructuring destructu
         if (TryConvertCompilerGeneratedType(value, type, destructuring, out var compilerGeneratedResult))
             return compilerGeneratedResult;
 
-        return new ScalarValue(value.ToString() ?? "");
+        var stringified = value.ToString();
+
+        if (stringified == null)
+            return ScalarValue.EmptyString; // ScalarValue.Null; ?
```

Comment:
🤔 ? Why so returning empty string? I tried to change it to return `ScalarValue.Null` but some tests failed. I have a suspicion that this was done by design a long time ago and now it can be dangerous to change behavior here. Honestly returning `null` here seems more reasonable.

Reply:
Is this even possible? I suppose some class could override `ToString` method, but that seems silly. As an outsider, returning null seems more logical.

Reply:
What I meant was, it would be silly for a class to override ToString to return `null`, but I'm sure it's being done somewhere for a reason I am not aware of.